metaflow/current.py

```diff
@@ -1,14 +1,13 @@
 from collections import namedtuple
 import os
-from typing import Any, Optional
+from typing import Any, Optional, TYPE_CHECKING
```

Comment:
note: for full disclosure, TYPE_CHECKING was introduced in 3.5.2. I feel it should still be OK and we can still support 3.5 (just 3.5.2+)